Images/map/in-person.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Where did these icons come from? I want to make sure people can't harass us about licensing (yes it's petty, but we have a target on our backs and it's better to avoid the headache).

Fallback choice might be to use the FA icons if needed.